pennylane/hf/tapering.py

```diff
@@ -246,7 +247,7 @@ def generate_symmetries(qubit_op, num_qubits):
         num_qubits (int): number of wires required to define the Hamiltonian
 
     Returns:
-        tuple (list[Hamiltonian], list[Observable]):
+        tuple (list[Hamiltonian], list[Operation]):
```

Comment:
What was the reason for this change? Was it just a typo from before or was the `generate_paulis` function updated?

Reply:
It was a minor typo.